webapp/handlers.py

```diff
@@ -57,6 +62,64 @@
     ["accept_encoding", "browser_family"],
 )
 
+CSP = {
+    "default-src": ["'self'"],
+    "img-src": [
+        "'self'",
+        "assets.ubuntu.com",
+        "res.cloudinary.com",
+        "dashboard.snapcraft.io",
+        "data: blob:",
+        "i.ytimg.com",
+        "i1.ytimg.com",
+        "i2.ytimg.com",
+        "i3.ytimg.com",
+        "i4.ytimg.com",
+        "www.googletagmanager.com",
+        "i.vimeocdn.com",
+        "forum.snapcraft.io",
+        "snapcraft.io",
+        # This is needed to allow images from
+        # https://www.google.*/ads/ga-audiences to load.
+        "*",
```

Comment:
you re right i wanted to keep them to avoid having to investigate which specific sources need to be added later but I can add those to the document so we have them ready when we decide to remove the wildcard.